node/api/statsv1alpha1/types.go

```diff
@@ -0,0 +1,345 @@
+/*
+Copyright 2015 The Kubernetes Authors.
```

Comment:
do we care this is a one-time copy and we lack automation to keep it up to date?

Reply:
No, we'll eventually move to use the package in k8s.io/kubelet.